src/main/drivers/transponder_ir.c

```diff
@@ -52,57 +56,29 @@ void transponderDMAHandler(dmaChannel_t* descriptor, dmaCallbackHandler_t* handl
     }
 }
 
-void transponderIrInit(void)
+void transponderIrInit(const uint8_t* transponderType)
```

Comment:
`transponderType` should be be called `transponderProvider` and should use values from a enum, perhaps `transponderProvider_e transponderProvider`  No need for a pointer to an int either unless I'm missing something.